spec/configure_helper_spec.rb

```diff
@@ -1,18 +1,41 @@
 require 'spec_helper.rb'
 
+shared_context 'with temp dir' do
+  before do
+    @tmpdir_path = Dir.mktmpdir
+    @pwd_before_spec_run = Dir.pwd
+    Dir.chdir(tmpdir_path)
+  end
+
+  after do
+    Dir.chdir(pwd_before_spec_run)
+    puts "tmp path is #{tmpdir_path}"
+    FileUtils.rm_rf(tmpdir_path)
+  end
+
+  attr_reader :tmpdir_path
+  attr_reader :pwd_before_spec_run
+end
+
 describe Fastlane::Helper::ConfigureHelper do
+  include_context 'with temp dir'
```

Comment:
What do you think of this? Worth following up with another PR that uses it in the other files that have a temp dir setup?

I'd extract it to it's own file, in that case